src/diffusers/utils/constants.py

```diff
@@ -48,7 +48,8 @@
     version.parse(importlib.metadata.version("transformers")).base_version
 ) >= version.parse(MIN_TRANSFORMERS_VERSION)
 
-USE_PEFT_BACKEND = _required_peft_version and _required_transformers_version
+_use_peft = _required_peft_version and _required_transformers_version
```

Comment:
I don't think I understand the change here - this should be equivalent to the code before, no?  cc @sayakpaul

Reply:
if you set it to False before creating the pipeline, you don't use the value of `_use_peft` anymore

```
import diffusers
diffusers.utils.USE_PEFT_BACKEND = False
```